packages/ui/package.json

```diff
@@ -89,7 +89,8 @@
     "tailwind-merge": "^2.3.0",
     "tailwindcss-animate": "^1.0.7",
     "vaul": "^0.9.1",
-    "zod": "^3.23.8"
+    "zod": "^3.23.8",
+    "zustand": "^4.5.4"
```

Comment:
`ui` package should not depend on `zustand`. Only `gitness` will be using zustand